scripts/sct_image.py

```diff
@@ -525,10 +546,10 @@ def concat_warp2d(fname_list, fname_warp3d, fname_dest):
 
 def multicomponent_split(im):
     """
-    Convert composite image (e.g., ITK warping field, 5dim) into several 3d volumes.
-    Replaces "c3d -mcs warp_comp.nii -oo warp_vecx.nii warp_vecy.nii warp_vecz.nii"
-    :param im:
-    :return:
+        Convert composite image (e.g., ITK warping field, 5dim) into several 3d volumes.
```

Comment:
any reason for adding this indentation?

Reply:
Nope! Changed in commit [4af3ca2](https://github.com/neuropoly/spinalcordtoolbox/commit/4af3ca27dbf6fb625c94f2750e3ac8e8297c2530)